### DISCUSSION TITLE
I'm attempting to add MIV to my OS [solved]

### DISCUSSION THREAD

**Discussion by winbamstudios** [CosmosOS/Cosmos#2388] [Q&A]
I'm attempting to add MIV to my OS lol thats it

**Comment by terminal-cs** [comment]
why have you opened this issue? you don't seem to have said any details about an error

**Comment by winbamstudios** [comment]
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0117	'Kernel' does not contain a definition for 'file'	Proxima64Sharp	C:\Users\[censored]\source\repos\Proxima64Sharp\Proxima64Sharp\MIV.cs	278	Active

**Comment by terminal-cs** [comment]
what is miv?

**Comment by winbamstudios** [comment]
https://github.com/bartashevich/MIV

**Comment by terminal-cs** [comment]
you need to create a string in your kernel class called file

**Comment by winbamstudios** [comment]
ohhhhhhhhhh
